Northstar.CustomServers/mod/scripts/vscripts/sh_progression.nut

```diff
@@ -88,21 +88,26 @@ bool function ClientCommand_SetProgression( entity player, array<string> args )
 
 /// Resets a specific Titan's Aegis rank back to `0`
 /// * `player` - The player entity to perform the action on
-/// * `args` - The arguments passed from the client command. `args[0]` should be an integer corresponding to the index of the Titan to reset.
+/// * `args` - The arguments passed from the client command. `args[0]` should be a string corresponding to the chassis name of the Titan to reset.
+/// * `chassis name` - Valid chassis are: Ion, Tone, Vanguard, Northstar, Ronin, Legion and Scorch.
```

Comment:
Not sure if persistence is case sensitive, but these are generally all lowercase.

Also since chassis name isnt a parameter to the function, it doesnt need the \* \` decorators. It can just be a continuation of the description of `args`

Reply:
Change done.